### PR TITLE
Audit logs blog

### PR DESCRIPTION
**Pull request by sean1588**
This PR is a simple blog about the audit logs feature that just got released on Tuesday. It is just a
quick post about the feature and a "how to" on how to access them.

### CODE REVIEW

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Getting Started with Pulumi Audit Logs"
+date: "2020-02-10"
+meta_desc: "Get started with audit logs"
+meta_image: "pulumi-webhooks.png"
+authors: ["sean-holung"]
+tags: []
```
Comment (sean1588) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]: Can I just add a tag here called "Audit Logs" or something similar, or is there like some predefined set of values I should be referencing?

Comment (joeduffy) [reply content/blog/getting-started-with-pulumi-audit-logs/index.md]: Maybe "enterprise"?

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Getting Started with Pulumi Audit Logs"
```
Comment (joeduffy) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]: Maybe we can make this pop a bit more for someone not already deep into Pulumi:

# Auditing Your Organization's Infrastructure as Code Activity

?

Comment (sean1588) [reply content/blog/getting-started-with-pulumi-audit-logs/index.md]: Yes, I like it. 👍

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+meta_desc: "Pulumi now supports Audit Logs. Learn how to audit your organization's infrastructure as code activity"
+meta_image: "auditlogs.png"
+authors: ["sean-holung"]
+tags: ["New-Features", "AuditLogs", "Enterprise"]
+---
+
+We are excited to announce the release of Audit Logs on
+[Pulumi](https://app.pulumi.com) for Enterprise organizations.
+Audit logs enable you to track the activity of users within an
+organization. Audit logs attempt to answer who did what, when
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
organization. They record who did what, when
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+they did it and where. They help answer these questions
+by recording entries for activities invoked by users.
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
by recording user actions.
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+Pulumi's audit logs allow you to account for the activity your
+users are taking within your organization. These logs are available to
+organizations with an Enterprise level subscription. The logs are immutable and
+are recorded whenever a user invokes an action. This allows the activity
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
and record all user actions. Auditing makes the activity
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md (hunk 3 above)
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
organization. They record who did what, when
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md (hunk 4 above)
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
by recording user actions.
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md (hunk 5 above)
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
record all user actions. Auditing makes the activity
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+of members in an organization to be traced.
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
of members in an organization attributable.
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+The logs are recorded with the UNIX timestamp of the event, the user
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
The logs capture the UNIX timestamp of the event, the user
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+who invoked the action, the event that took place, and the source IP
+of the call being made.
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
of the call the user made.
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+## Using Pulumi Audit Logs
+
+Audit logs are available to organizations with an Enterprise level subscription
+only. You can view the audit logs for your organization if you
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
only. If you are an organization administrator, you can view
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+are an organization admin, by selecting your organization from the
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
your organization's audit logs by selecting your organization from the
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+organization drop down. Then click on the settings tab. On the left nav-bar
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
organization dropdown. Then click on the settings tab. On the left nav-bar
```

File: content/blog/getting-started-with-pulumi-audit-logs/index.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+you should see a tab called Audit Logs. Clicking here will allow you to view
+the most recent audit logs for your organization.
+
+![auditlogs](./auditlogs.png)
+
+This will show the most recent events in decending order. You can
+also filter logs by a particular user by clicking on the user's avatar. This will
```
Comment (spara) [inline content/blog/getting-started-with-pulumi-audit-logs/index.md]:
```suggestion
also filter logs by a particular user by clicking on the user's avatar. Doing so
```

**Pull request merged**